yarn.lock

```diff
@@ -5175,6 +5175,11 @@ color@3.0.x:
     color-convert "^1.9.1"
     color-string "^1.5.2"
 
+colorette@^1.1.0:
```

Comment:
Yes - needed to change rollup-plugin-copy-alias to rollup-plugin-copy so that only one instance of the protos is copied (rollup-plugin-copy-alias doesn't let you specify the destination)